### PR TITLE
Update proc_creation_win_taskkill_execution.yml

### PR DESCRIPTION
**Pull request by MalGamy12**
### Summary of the Pull Request
Add command line argument flags used with taskkill by the attacker to terminate a process by its PID
### Changelog
update: Process Terminated Via Taskkill - Add `/pid` flag and windash support
### Example Log Event

``` Process Create:
RuleName: -
UtcTime: 2024-10-02 21:02:05.228
ProcessGuid: {6e6be129-b4cd-66fd-cb01-000000001900}
ProcessId: 4568
Image: C:\Windows\System32\taskkill.exe
FileVersion: 10.0.19041.1 (WinBuild.160101.0800)
Description: Terminates Processes
Product: Microsoft® Windows® Operating System
Company: Microsoft Corporation
OriginalFileName: taskkill.exe
CommandLine: taskkill  /pid 380 /f
CurrentDirectory: C:\Users\MalGamy\
User: DESKTOP-0TOC207\MalGamy
LogonGuid: {6e6be129-ae65-66fd-58dc-0d0000000000}
LogonId: 0xDDC58
TerminalSessionId: 1
IntegrityLevel: Medium
Hashes: SHA256=56F8CC2C1790C389394733B84C3FB55E10977E9F0FE0C08110AC11F0FE47F05E
ParentProcessGuid: {6e6be129-b4b0-66fd-c901-000000001900}
ParentProcessId: 1416
ParentImage: C:\Windows\System32\cmd.exe
ParentCommandLine: "C:\Windows\system32\cmd.exe" 
ParentUser: DESKTOP-0TOC207\MalGamy 
```

### CODE REVIEW

File: rules-threat-hunting/windows/process_creation/proc_creation_win_taskkill_execution.yml [hunk 1]
```diff
@@ -20,16 +21,18 @@ detection:
...
         - OriginalFileName: 'taskkill.exe'
-    selection_cli:
-        CommandLine|contains|all:
-            - ' /f'
+    selection_1_cli:
+        CommandLine|contains: ' /f'
+    selection_2_cli:
+        CommandLine|contains:
             - ' /im '
+            - ' /pid '
```
Comment (frack113) [inline rules-threat-hunting/windows/process_creation/proc_creation_win_taskkill_execution.yml:29]: Do want `/f AND (im OR pid)` or `/f OR im OR pid` ?
Ref doc https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/taskkill
there is a `/fi` for filter, so if we want only the forcefully  we need to write ` /f ` but the problem is that you just have to put the /f at the end to bypass the rule ...

Comment (MalGamy12) [reply rules-threat-hunting/windows/process_creation/proc_creation_win_taskkill_execution.yml:29]: i want ```/f AND (im OR pid)```.

Comment (frack113) [reply rules-threat-hunting/windows/process_creation/proc_creation_win_taskkill_execution.yml:29]: So it is more `condition: all of selection_* and not 1 of filter_main_*`

**Review by nasbench — APPROVED** [review]

**Review by frack113 — APPROVED** [review]
LGTM

**Pull request merged**